src/main/kotlin/no/item/gradle/InitRunnerTask.kt

```diff
@@ -0,0 +1,22 @@
+package no.item.gradle
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+import javax.inject.Inject
+
+open class InitRunnerTask @Inject constructor(private val extension: InitRunnerExtension) : DefaultTask(){
```

Comment:
If this task is listing xml-files, maybe we should rename the file (and class)  to `FindXmlFilesTask.kt`?

Reply:
Good idea, implemented